interp/src/flatten/structures/environment/program_counter.rs

```diff
@@ -56,6 +56,67 @@ impl ControlPoint {
             false
         }
     }
+
+    /// Returns a string showing the path from the root node to input node.
+    /// How to get context?
```

Comment:
remove `/// How to get context?` also specify that the path is displayed in the
minimal metadata path syntax